/**:
  ros__parameters:
    base_frame_id: base_link
    map_frame_id: map

    matching_score_threshold: 1.5
    matching_fail_num_threshold: 3

    history_size: 20
    mahalanobis_threshold: 10.0

    scan_voxel_size: 2.0
    map_voxel_size: 0.2
    crop_min_x: -1.0
    crop_max_x: 1.0
    crop_min_y: -1.0
    crop_max_y: 1.0
    crop_min_z: -1.0
    crop_max_z: 1.0

    transformation_epsilon: 0.01
    step_size: 0.1
    resolution: 2.0
    max_iteration: 30
    omp_num_thread: 4

